includes/notes/data/class-wc-calypso-bridge-admin-note-data-store.php

```diff
@@ -35,6 +34,27 @@ protected function has_allow_list() {
 		return wc_calypso_bridge_is_ecommerce_trial_plan();
```

Comment:
I changed this to return `true` to test this on my Woo Express site.

Reply:
Good call!